rollingmean.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+import config
+import csv
+
+def rolling_mean(file, columnData, window_size):
+
+    current = 1
+    first_run = True
+    for i in range(1, len(columnData["Time"])):
+        if int(columnData["Time"][i]) == 1:
+            df = pd.read_csv(file, skiprows=current, nrows=i-current, names = ["Time", "Drive",	"Stimulus", "Failure", "Palm.EDA", "Heart.Rate", "Breathing.Rate",
```

Comment:
Big space after "Drive", doesn't need to be there

Reply:
done